index.js

```diff
@@ -169,12 +169,20 @@ module.exports = function (urlRoot, pathRoot) {
           req, filePath, fileType: 'js', config
         });
         delete require.cache[require.resolve(path.resolve(filePath))];
-        const customMiddleware = require(path.resolve(filePath));
+        let customMiddleware = require(path.resolve(filePath));
         if (requestParams) {
           req.params = requestParams;
         }
         bodyParser.json()(req, res, () => {
-          customMiddleware(req, res, next);
+          if (typeof customMiddleware === 'function') {
```

Comment:
Maybe flip this case... If the customMiddleware exports not a function or array, you'll get an exception somewhere lower I guess? So instead of checking if function, check if not array?

Reply:
Actually, if someone exports something then function or array of functions, this will fail for sure. But I'm not sure, should I try to handle this.

Reply:
Let's postpone this improvement as a separate thing. :)